### PR TITLE
Scaling

### PR DESCRIPTION
**Pull request by DonDuala**

### CODE REVIEW

File: src/scalper.ts [hunk 1]
```diff
@@ -610,7 +610,10 @@ export class Scalper {
 
     // Settle Funds
     try {
-      await settleOpenBook(this.connection, this.owner, spotMarket, this.symbol, 'USDC');
+      const settleTransaction = await settleOpenBook(this.connection, this.owner, spotMarket, this.symbol, 'USDC');
+      if (settleTransaction != undefined) {
```
Comment (brittcyr) [inline src/scalper.ts:614]: add a comment explaining what an undefined response means. does undefined mean that it was a success or not?

Comment (brittcyr) [reply src/scalper.ts:614]: ideally, we dont rely on undefined and have an explicit success or failure value

File: src/config.ts [hunk 2]
```diff
@@ -4,6 +4,7 @@ import { DIPDeposit } from './common';
 
 export const IS_DEV = process.env.DEV != 'false';
 export const API_URL = process.env.RPC;
+export const PRIORITY_FEE = parseInt(process.env.FEE);
```
Comment (brittcyr) [inline src/config.ts:7]: we should start making params like this one be optional with a default value.

**Review by brittcyr — APPROVED** [review]

File: src/scalper.ts (hunk 1 above)
Comment (DonDuala) [reply src/scalper.ts:614]: Only returns a tx instruction if there are funds to settle, added comment

**Pull request merged**